internal/provider/resource_cloudexport.go

```diff
@@ -88,6 +90,6 @@ func resourceCloudExportDelete(ctx context.Context, d *schema.ResourceData, m in
 	if err != nil {
 		return detailedDiagError("Failed to delete cloud export", err)
 	}
-	tflog.Debug(ctx, "Deleted cloud export Kentik API response")
+	tflog.Debug(ctx, fmt.Sprintf("Deleted cloud export with ID %v in Kentik", d.Get("id").(string)))
```

Comment:
Please use the API of Debug() function fully, just like in other places (e.g. L53), instead of using Sprintf.

Reply:
Done